### DISCUSSION TITLE
ERR Cannot retrieve the ACME challenge for [domain]

### DISCUSSION THREAD

**Discussion by seb-L8L** [coollabsio/coolify#4281] [Q&A]
We have a recurring error message in the Proxy logs: "ERR Cannot retrieve the ACME challenge for [domain] (token "[token]") providerName=acme"
Anyone got an idea why?
Thanks

**Comment by MatteoGauthier** [comment]
Same issue even if the domain mapping is working 🤷

**Comment by rootacc3ss** [comment]
Bump. Getting this persistently on self-hosted v4.

**Comment by djsisson** [comment]
if you are seeing this message, either

1) you have turned on a dns proxy like cloudflare
2) you have blocked port 80
3) you are being rate limited
4) it is an old subdomain that had a certificate and is expiring but you now no longer use it

for option one you need to switch to using dns challenge or install certificates manually

**Reply by rootacc3ss** [reply]
it didn't seem to be any of that! i reset my instance and deleted the deployment that was having issues with it, now no issues at all
